sdk/peering/Microsoft.Azure.Management.Peering/src/Microsoft.Azure.Management.Peering.csproj

```
@@ -7,13 +7,11 @@
 		<PackageId>Microsoft.Azure.Management.Peering</PackageId>
 		<Description>Microsoft Azure Management Peering library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Peering</AssemblyName>
-		<Version>2.0.0-preview</Version>
+		<Version>2.0.1-preview</Version>
 		<PackageTags></PackageTags>
-    <PackageReleaseNotes><![CDATA[
-		- Added Registered ASN operations.
-		- Added Registered Prefix operations.
-		- Added Improvement to Peering Service operations. 
-		]]>
+    <PackageReleaseNotes>
+		- Made microsoft session address non-readonly
+		
```

Please keep the CDATA tag
```suggestion
    <PackageReleaseNotes><![CDATA[
- Made microsoft session address non-readonly
]]>
```